receiver/riakreceiver/integration_test.go

```diff
@@ -87,5 +87,5 @@ func TestRiakIntegration(t *testing.T) {
 
 	actualMetrics := consumer.AllMetrics()[0]
 
-	pmetrictest.CompareMetrics(expectedMetrics, actualMetrics, pmetrictest.IgnoreMetricValues())
+	pmetrictest.CompareMetrics(expectedMetrics, actualMetrics, pmetrictest.IgnoreMetricValues()) //nolint:errcheck
```

Comment:
Why not expect no error?

Reply:
I think you mentioned something in the description. Fine to resolve in a separate PR.

Reply:
You're correct that it should be that way, but see issue description. I've captured each in an issue for now.